File: 24_10/16/P12982/Solution.java
import java.util.*;

class Solution {
    public int solution(int[] d, int budget) {
        // 1. 가격표 정렬
        Arrays.sort(d);
        
        // 2. 낮은 가격의 신청금부터 합산
        int count = 0;
        for (int cost : d) {
            if (budget < cost) break;
            budget -= cost;
            count++;
        }
        
        return count;
    }
}

File: 24_10/16/P12982/solution.js
function solution(d, budget) {
    // 1. 가격 정렬
    d.sort((a,b) => a - b);
    
    // 2. 최소 가격부터 탐색
    let answer = 0;
    for (const cost of d) {
        if (budget < cost) break;
        budget -= cost;
        answer++;
    }

    return answer;
}